templates/contest/edit.html

```diff
@@ -24,6 +24,32 @@
             $('#form_set tr').formset({
                 prefix: '{{ contest_problem_formset.prefix }}'
             });
+
+            $(document).on('keyup', '#id_private_contestants + .select2 .select2-search__field', function (e) {
+                if (e.keyCode === 13) {
```

Comment:
If we use this, the user might don't know that they need to press the enter since the form doesn't say to press the enter.

Perhaps we could listen to the Paste event?
![image](https://user-images.githubusercontent.com/23715841/131328884-aef2d3fa-bc31-42df-85b5-8c14087820bb.png)

Reply:
We can change "No results found" to something like "Please press Enter to select contestants..."

Reply:
Sounds ok to me